src/contracts/interfaces/IRwaAToken.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IRwaAToken
+ * @author Aave
+ * @notice Defines the basic interface for an RwaAToken.
+ */
+interface IRwaAToken {
+  /**
+   * @dev The role required to perform authorized transfers of RWA aTokens
+   * This role should be granted in the ACLManager
+   */
+  function ATOKEN_TRANSFER_ROLE() external pure returns (bytes32);
+
+  /**
+   * @notice ERC-20 Permits are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function permit(
+    address owner,
+    address spender,
+    uint256 value,
+    uint256 deadline,
+    uint8 v,
+    bytes32 r,
+    bytes32 s
+  ) external;
+
+  /**
+   * @notice ERC-20 Approvals are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function approve(address spender, uint256 amount) external returns (bool);
+
+  /**
+   * @notice ERC-20 Allowances are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function increaseAllowance(address spender, uint256 addedValue) external returns (bool);
+
+  /**
+   * @notice ERC-20 Allowances are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function decreaseAllowance(address spender, uint256 subtractedValue) external returns (bool);
+
+  /**
+   * @notice ERC-20 Transfers are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function transfer(address recipient, uint256 amount) external returns (bool);
+
+  /**
+   * @notice ERC-20 Transfers are not supported for RWA aTokens.
```

Comment:
i d remove the `ERC-20` and make it general, for all functions

Reply:
[0fc2dad](https://github.com/aave/aave-v3-horizon/pull/1/commits/0fc2dad326889c78fa2cd968375e0ef98f12989a)